lightning/src/ln/channel.rs

```diff
@@ -8363,7 +8836,9 @@ impl<'a, 'b, 'c, ES: Deref, SP: Deref> ReadableArgs<(&'a ES, &'b SP, u32, &'c Ch
 				channel_keys_id,
 
 				blocked_monitor_updates: blocked_monitor_updates.unwrap(),
-			}
+			},
+			#[cfg(dual_funding)]
+			dual_funding_channel_context: None,
```

Comment:
Is the `dual_funding_channel_context` field never persisted?

Reply:
It's not persisted yet in this PR. Will be important for RBF.